src/string/HslColor.ts

```diff
@@ -3,7 +3,15 @@
  *
  * @since 0.0.3
  * @example
- *   background-color: hsl(270 60% 70% / 0.7);
+ *   const hue = 270
+ *   const saturation = 60
+ *   const lightness = 70
+ *   const alpha = 0.7
+ *
+ *   const styles = styled.div`
```

Comment:
🔧  unfortunately (or possibly fortunately) docs-ts will actually try and compile all examples, but here we don't have styled listed as a dependency, so it won't typecheck because `styled` is undefined.  And this is why CI is failing

Reply:
Ah, ok! Weird. I'll try building the docs locally first next time 😅